docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     env_file:
       - ./.docker/local.env
     ports:
-      - '3308:3306'
+      - '3306:3306'
```

Comment:
This was a stumbling block that I couldn't see but @kschelonka did during our pairing. Thank you Kat!